packages/ui-extensions/docs/surfaces/point-of-sale/reference/examples/tile/default.example.ts

```diff
@@ -0,0 +1,20 @@
+import {
+  Tile,
+  extension,
+} from '@shopify/ui-extensions/point-of-sale';
+
+export default extension(
+  'pos.home.tile.render',
+  (root, api) => {
+    const tile = root.createComponent(Tile, {
+      title: 'My app',
+      subtitle: 'Hello world!',
+      enabled: true,
+      onPress: () => {
+        api.smartGrid.presentModal();
```

Comment:
```suggestion
        api.action.presentModal();
```

Aren't we are deprecating smartGrid?